plugins/aws/eventbridge/eventBusCrossAccountAccess.js

```diff
@@ -0,0 +1,135 @@
+var helpers = require('../../../helpers/aws');
+var async = require('async');
+
+module.exports = {
+    title: 'Event Bus Cross Account Access',
+    category: 'EventBridge',
+    description: 'Ensure that event bus is configured to allow access to whitelisted AWS account principals.',
+    more_info: 'Event bus policy should be configured to allow access only to whitelisted/trusted cross-account principals.',
+    link: 'https://docs.amazonaws.cn/en_us/eventbridge/latest/userguide/eb-event-bus-perms.html',
+    recommended_action: 'Configure event bus policies that allow access to whitelisted/trusted cross-account principals.',
+    apis: ['EventBridge:listEventBuses', 'STS:getCallerIdentity', 'Organizations:listAccounts'],
+    settings: {
+        whitelisted_aws_account_principals: {
+            name: 'Whitelisted AWS Account Principals',
+            description: 'A comma-separated list of trusted cross account principals',
+            regex: '^.*$',
+            default: ''
+        },
+        whitelisted_aws_account_principals_regex: {
+            name: 'Whitelisted AWS Account Principals Regex',
+            description: 'If set, plugin will compare cross account principals against this regex instead of otherwise given comma-separated list' +
+                'Example regex: ^arn:aws:iam::(111111111111|222222222222|):.+$',
+            regex: '^.*$',
+            default: ''
+        },
+        iam_whitelist_aws_organization_accounts: {
+            name: 'Whitelist AWS Organization Accounts',
+            description: 'If true, trust all accounts in current AWS organization',
+            regex: '^(true|false)$',
+            default: 'false'
+        }
```

Comment:
```suggestion
        eventbridge_whitelisted_aws_account_principals: {
            name: 'EventBridge Whitelisted AWS Account Principals',
            description: 'A comma-separated list of trusted cross account principals',
            regex: '^.*$',
            default: ''
        },
        whitelisted_aws_account_principals_regex: {
            name: 'Whitelisted AWS Account Principals Regex',
            description: 'If set, plugin will compare cross account principals against this regex instead of otherwise given comma-separated list' +
                'Example regex: ^arn:aws:iam::(111111111111|222222222222|):.+$',
            regex: '^.*$',
            default: ''
        },
        iam_whitelist_aws_organization_accounts: {
            name: 'Whitelist AWS Organization Accounts',
            description: 'If true, trust all accounts in current AWS organization',
            regex: '^(true|false)$',
            default: 'false'
        }
```